examples/cogvideo/train_cogvideox_lora.py

```diff
@@ -866,7 +866,14 @@ def prepare_rotary_positional_embeddings(
     return freqs_cos, freqs_sin
 
 
-def get_optimizer(args, params_to_optimize):
+def get_optimizer(accelerator, args, params_to_optimize):
+    # Use DeepSpeed optimzer
+    if (
+        accelerator.state.deepspeed_plugin is not None
+        and "optimizer" in accelerator.state.deepspeed_plugin.deepspeed_config
+    ):
+        return DummyOptim(params_to_optimize, lr=args.learning_rate)
```

Comment:
@bghira I believe this is the correct usage?

Reply:
indeed

Reply:
```py
    if use_deepspeed_optimizer:
        optimizer_class = accelerate.utils.DummyOptim
        extra_optimizer_args["lr"] = float(args.learning_rate)
        extra_optimizer_args["betas"] = (args.adam_beta1, args.adam_beta2)
        extra_optimizer_args["eps"] = args.adam_epsilon
        extra_optimizer_args["weight_decay"] = args.adam_weight_decay


    if use_deepspeed_scheduler:
        lr_scheduler = accelerate.utils.DummyScheduler(
            optimizer,
            total_num_steps=args.max_train_steps,
            warmup_num_steps=args.lr_warmup_steps,
        )
```